compiler/arm/codegen/TreeEvaluatorTable.hpp

```diff
@@ -119,6 +119,7 @@
    TR::TreeEvaluator::unImpOpEvaluator,     // TR::brem
    TR::TreeEvaluator::unImpOpEvaluator,     // TR::srem
    TR::TreeEvaluator::iremEvaluator,        // TR::iurem
+   TR::TreeEvaluator::lremEvaluator,        // TR::lurem
```

Comment:
I think this should be left as `TR::TreeEvaluator::unImpOpEvaluator`.  I don't believe the ARM long division/remainder helpers can handle unsigned operations.

I will be creating an issue shortly to complete the implementation of unsigned arithmetic on all code generators and I will include this particular case as part of that.